DevDynamo.Web/Areas/ApiV1/Models/SystemResponse.cs

```diff
@@ -0,0 +1,13 @@
+﻿using DevDynamo.Models;
+using System.ComponentModel.DataAnnotations;
+
+namespace DevDynamo.Web.Areas.ApiV1.Models
+{
+  public class SystemResponse
+  {
+    public string? version { get; set; }
+    public string? environment { get; set; }
+    public string? now { get; set; }
```

Comment:
We should name properties with CamelCase as it is a C# code. When it has returned from the API, it'll be converted to camelCase automatically.